run/image-processing/.env

```diff
@@ -0,0 +1,2 @@
+export INPUT_BUCKET_NAME=adamross-svls-kibble
```

Comment:
I'm assuming this will change.

Reply:
Hm, something happened to my global gitignore that this file got in. Will remove.